test/e2e/egressip.go

```diff
@@ -1733,4 +1802,536 @@ spec:
 			framework.Failf("unexpected server IP route cache: %s", stdout)
 		}
 	})
+
+	/* This test does the following:
+	   Note: 'OVN managed network' here means that OVN directly controls an interface that is attached to a network. This is
+	   accomplished with OVN and therefore ovs rules. 'Non-OVN managed network' means that we partly use OVN/ovs rules to route the packet
+	   and also use the linux networking stack to perform the local host routing when the packet is expected to egress that
+	   particular node.
+
+	   0. Set two nodes as available for egress
```

Comment:
great comments!

Reply:
Thanks - I have a local commit to add chaos testing in a later PR :) I think youll enjoy it.